bears/documentation/DocumentationStyleBear.py

```diff
@@ -0,0 +1,87 @@
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+from coalib.bearlib.languages.documentation.DocstyleDefinition import (
+    DocstyleDefinition)
+from coalib.bearlib.languages.documentation.DocumentationExtraction import (
```

Comment:
Hmm, so for this to work I have to also modify the `__init__` file to import stuff beforehand, would that be okay?